### PR TITLE
Add new paper for awesome-pets/psi

### PR DESCRIPTION
**Pull request by tarantula-leo**
**Type of change**

- [x] Add new papers (Please tell us why you think this paper is awesome!)
- [ ] Fix the category of an existing paper/papers (Please tell us the reasons)
- [ ] Add a new tool/primitive/application with a new markdown page (Thank you! Also, please tell us more about this awesome thing!)

**Description**

> Please include a summary of the change and the relevant motivation. 

Add a new EuroCrypt20 paper: PSI from PaXoS: Fast, Malicious Private Set Intersection. This paper first used a new data structure, called a probe-and-XOR of strings(PaXoS) in PSI.

### CODE REVIEW

**Comment by tarantula-leo** [comment]
[@jamie-cui](https://www.github.com/jamie-cui)

File: docs/awesome-pets/papers/applications/set/psi.md [hunk 1]
```diff
@@ -53,13 +53,18 @@ Note: one paper may be included in several categories (e.g. a paper may introduc
   *Gayathri Garimella, Payman Mohassel, Mike Rosulek, Saeed Sadeghian, Jaspal Singh*  
   PKC 2021, [eprint](https://eprint.iacr.org/2021/243), GMRS21
   
+- PSI from PaXoS: Fast, Malicious Private Set Intersection
```
Comment (Jamie-Cui) [inline docs/awesome-pets/papers/applications/set/psi.md:56]: Can you add two spaces at the end of this line? it'll look nicer on github

File: docs/awesome-pets/papers/applications/set/psi.md [hunk 2]
```diff
@@ -53,13 +53,18 @@ Note: one paper may be included in several categories (e.g. a paper may introduc
...
+  *Benny Pinkas, Mike Rosulek, Ni Trieu, Avishay Yanai*
```
Comment (Jamie-Cui) [inline docs/awesome-pets/papers/applications/set/psi.md:57]: Same here

**Review by Jamie-Cui — CHANGES_REQUESTED** [review]

**Comment by Jamie-Cui** [comment]
Thanks for your contribution~ b.t.w. you can also add your github id here: https://github.com/secretflow/secretflow/blob/main/docs/awesome-pets/awesome-pets.md

image

File: docs/awesome-pets/papers/applications/set/psi.md (hunk 2 above)
Comment (tarantula-leo) [reply docs/awesome-pets/papers/applications/set/psi.md]: I modified it here(in my branch).
image
Is that going to show up here?

Comment (Jamie-Cui) [reply docs/awesome-pets/papers/applications/set/psi.md]: > Is that going to show up here?

LGTM, I'm ok with its position

**Review by Jamie-Cui — CHANGES_REQUESTED** [review]
LGTM

**Review by Jamie-Cui — APPROVED** [review]
LGTM

**Pull request merged**